pgmpy/estimators/HillClimbSearch.py

```diff
@@ -158,6 +158,8 @@ def estimate(self, start=None, tabu_length=0, max_indegree=None):
 
             if best_operation is None:
                 break
+            if best_score_delta < epsilon:
```

Comment:
No, i mean something like x_curr - x_prev where x_curr indicates the current value of the score of the system. We will keep the values of score of the model in iteration i and iteration i-1 if the score dosen't change much then we can sure that we have reached some local maxima. This is a standard technique in ML